tests/conftest.py

```diff
@@ -22,7 +22,7 @@ def pytest_report_header(config):
 
 
 @pytest.fixture
-def request_schema():
+def request_schema_fixture():
```

Comment:
I don't think its a good idea to add postfix `fixture` to fixtures. More than that such changes are not related to our issue.

Reply:
Hm, sorry, I see that you did it because of naming conflict between fixtures and decorators names.